app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb

```diff
@@ -1,16 +1,19 @@
-<p>You may be eligible for a bursary or scholarship if your teacher training course leads to <%= link_to("qualified teacher status (QTS)", page_path("train-to-be-a-teacher/what-is-qts")) %>. These are tax-free amounts of money you receive to train in certain subjects. You do not need to pay them back.</p>
-
-<p>Your bursary or scholarship will be paid in a minimum of 10 equal monthly instalments over the duration of your course by your teacher training provider.</p>
-
-<p>Your teacher training provider will confirm the payment dates and amounts.</p>
-
-<p>The following amounts apply if you're starting your teacher training course between September 2024 and July 2025.</p>
+<section class="col col-720">
+  <p>This page is for people who have a degree or are studying towards one, and want to do postgraduate teacher training in England.  To help fund your training, you may be eligible for a bursary or scholarship.</p>
+  <h2>What's a bursary?</h2>
```

Comment:
I think the heading structure is very confusing as well - they don't feel very mutually exclusive/clear, which would be hard work for screen reader users

- and it's a bit confusing to have headings of
* **What's a bursary**
* **What's a scholarship**
* What to consider
* What funding could I get for teacher training/
* **Bursaries**
* **Scholarships**
* **Compare bursaries and scholarships**
* If you think you are not eligible